addons/test_sale_product_configurators/tests/test_event_sale_with_product_configurator.py

```diff
@@ -0,0 +1,133 @@
+            'name': 'Registration Event (TEST variants)',
+            'list_price': 30.0,
+            'detailed_type': 'event',
+        })
+
+        # Generate variants
+        cls.env['product.template.attribute.line'].create([{
+            'product_tmpl_id': cls.event_product_template.id,
+            'attribute_id': product_attribute_age.id,
+            'value_ids': [
+                (4, product_attribute_value_0.id),
+                (4, product_attribute_value_1.id),
+                (4, product_attribute_value_2.id),
+                (4, product_attribute_value_3.id),
+            ],
+        }])
+
+        # Apply a price_extra for the attributes
+        cls.event_product_template.attribute_line_ids[0].product_template_value_ids[0]. \
+            price_extra = -20.00
+        cls.event_product_template.attribute_line_ids[0].product_template_value_ids[2]. \
+            price_extra = -10.00
+        cls.event_product_template.attribute_line_ids[0].product_template_value_ids[3]. \
+            price_extra = 30.00
+
+        # Create the event and link it to the product variants as event tickets
+        cls.event = cls.env['event.event'].create({
+            'name': 'TestEvent',
+            'auto_confirm': True,
+            'date_begin': fields.Datetime.to_string(datetime.today() + timedelta(days=1)),
+            'date_end': fields.Datetime.to_string(datetime.today() + timedelta(days=15)),
+            'date_tz': 'Europe/Brussels',
+        })
+
+        for variant in cls.event_product_template.attribute_line_ids[0].product_template_value_ids:
+            cls.env['event.event.ticket'].create({
+                'name': variant.name,
+                'event_id': cls.event.id,
+                'product_id': variant.ptav_product_variant_ids[0].id,
+            })
+            if variant.name != 'VIP':
+                cls.env['event.event.ticket'].create({
+                    'name': variant.name + ' + meal',
+                    'event_id': cls.event.id,
+                    'product_id': variant.ptav_product_variant_ids[0].id,
+                    'price': variant.ptav_product_variant_ids[0].lst_price + 5,
+                })
+
+            # Adding an optional product
+            cls.product_product_memorabilia = cls.env['product.template'].create({
+                'name': 'Memorabilia (TEST)',
+                'list_price': 16.50,
+            })
+
+            cls.event_product_template.optional_product_ids = [cls.product_product_memorabilia.id,]
+
+    @users('salesman', 'salemanager', 'admin')
+    def test_event_using_product_configurator(self):
+        self.start_tour(
+            "/web", 'event_sale_with_product_configurator_tour', login=self.env.user.login
+        )
+        partner = self.env['res.partner'].search([('name', '=', 'Tajine Saucisse')])
+        sale_order = self.env['sale.order'].search(
+            [('partner_id', 'ilike', partner.id), ('create_uid', "=", self.env.user.id)]
```

Comment:
```suggestion
            [('partner_id', '=', partner.id), ('create_uid', "=", self.env.user.id)]
```
Is the ilike necessary ? Prefer simple strict comparison when it works :)